android/app/build.gradle

```diff
@@ -204,7 +204,7 @@ dependencies {
     implementation project(':realm')
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.android.support:appcompat-v7:26.1.0"
-    implementation "com.facebook.react:react-native:+"  // From node_modules
+    compile ("com.facebook.react:react-native:0.53.3") { force = true }  // From node_modules
```

Comment:
Can you add a comment detailing what this does? Also not clear if it's a workaround or the proper fix.

Reply:
Comment added. It's a proper fix because we don't rely on gradle dependency resolution, but explicitly set version we want use.